src/Routers/FeaturesRouter.js

```diff
@@ -20,10 +20,10 @@ export class FeaturesRouter extends PromiseRouter {
     if (!req.auth.isMaster) {
       return masterKeyRequiredResponse();
     }
-    
+
     return Promise.resolve({
       response: {
-        results: [getFeatures()]
+        results: getFeatures()
```

Comment:
That is more important in eg. `/schemas` where you could legitimately create a class named `code` and another named `message`